keylime/mba/elchecking/example.py

```diff
@@ -303,6 +302,20 @@ def vmlinuz_test(kernel: typing.Dict[str, str]) -> tests.Test:
                 ),
             ),
         )
+        # edk2 measures up to 4 of those events, where we do not have a good way to get a reference
+        # See:
+        # - https://github.com/keylime/keylime/issues/1393
+        # - https://github.com/tianocore/edk2/commit/935343cf1639a28530904a1e8d73d6517a07cbff
+        dispatcher.set(
+            (1, "EV_PLATFORM_CONFIG_FLAGS"),
```

Comment:
Is this PCR1 ?

Reply:
Yes it is. See the `TpmMeasureAndLogData` function call in the referenced commit https://github.com/tianocore/edk2/commit/935343cf1639a28530904a1e8d73d6517a07cbff